controllers/openstacknetconfig_controller.go

```diff
@@ -529,17 +554,26 @@ func (r *OpenStackNetConfigReconciler) applyNetConfig(
 
 	op, err := controllerutil.CreateOrUpdate(context.Background(), r.Client, osNet, apply)
 	if err != nil {
-		err = common.WrapErrorForObject(fmt.Sprintf("Updating %s OpenStackNetwork", osNet.Name), osNet, err)
+		cond.Message = fmt.Sprintf("Failed to create or update %s %s ", osNet.Kind, osNet.Name)
+		cond.Reason = ospdirectorv1beta1.ConditionReason(ospdirectorv1beta1.OpenStackProvisionServerCondReasonCreateError)
```

Comment:
More copy-paste left-overs? (`OpenStackProvisionServerCondReasonCreateError`)

Reply:
yes, good catch. will fix it

Reply:
fixes in latest commit